foundation_cms/templates/patterns/pages/nothing_personal/home_page.html

```diff
@@ -19,9 +19,12 @@
     {% if page.hero_item %}
       {% include "patterns/components/nothing_personal/home_page/hero_item.html" with hero_item=page.hero_item orientation=page.hero_item_orientation %}
     {% endif %}
-    {% if page.featured_items.all %}
-      {% include "patterns/components/nothing_personal/home_page/featured_items.html" with featured_items=page.featured_items.all %}
+
+    {% get_localized_featured_items page as localized_items %}
+    {% if localized_items %}
+      {% include "patterns/components/nothing_personal/home_page/featured_items.html" with featured_items=localized_items %}
     {% endif %}
+
```

Comment:
```suggestion

    {% if localized_featured_pages %}
      {% include "patterns/components/nothing_personal/home_page/featured_items.html" with featured_items=localized_featured_pages %}
    {% endif %}

```

If my suggestion sounds good to you, we can remove the use of of the template tag here and just update the `if` and `include` statement like so ^

Lastly, we would need to make a minor change to `featured_items.html`:

Instead of:
```jinja2
{% for featured_rel in featured_items %}
  {% if featured_rel and featured_rel.page %}
    {% with featured_item=featured_rel.page.specific %}
...
    {% endwith %}
  {% endif %}
{% endfor %}
```

we can wrap the template contents like so: 
```jinja2
{% for featured_item in featured_items %}
...
{% endfor %}
```